### PR TITLE
Refactor the QA Raw Command

### LINKED ISSUE #3214: [Bug] `stocks/qa/raw --descend` does not sort
**Describe the bug**

**To Reproduce**
Steps(from the start) and commands to reproduce the behavior

**Screenshots**
If you are running the terminal using the conda version please
rerun the terminal with `python terminal.py --debug`, and then
recreate your issue. Then include a screenshot of the entire 
error printout.

**Desktop (please complete the following information):**

- OS: [e.g. Mac Sierra]
- Python version [e.g. 3.6.8]

**Additional context**
Add any other information that you think could be useful for us.

### PR DESCRIPTION
**Pull request by colin99d**
# Description

Fixes #3214

This command shows the entire dataframe instead of one row. It also implements the sortby argument.

# How has this been tested?

* Please describe the tests that you ran to verify your changes.
* Provide instructions so we can reproduce.
* Please also list any relevant details for your test configuration.
- [x] Make sure affected commands still run in terminal
    - `python terminal.py "stocks/load gme/qa/raw -s Open --descend" `
    - `python terminal.py "stocks/load gme/qa/raw -s OpEn --descend"`
    - `python terminal.py "stocks/load gme/qa/raw -s loGprIcE"`
    - `python terminal.py "stocks/load gme/qa/raw -s logprice"`
    - `python terminal.py "crypto/load btc/qa/raw -s Open --descend"`
    - `python terminal.py "etf/load voo/candle --sort Open --descending --raw"`
    - `python terminal.py "forex/load USDGBP/qa/raw --sort Open --descend"`
    - `python terminal.py "economy/macro/qa/raw -s united_states_cpi"`
    - `python terminal.py "stocks/load cano/candle --sort High --descending --raw"`
- [x] Ensure the SDK still works
    - This function in not on the SDK (its just shows a df, so it wouldnt make much sense to have it).
    - `openbb.stocks.candle("TSLA", raw=True)`
    - `openbb.etf.candle("VOO", raw=True)`

# Checklist:

- [x] Update [our Hugo documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).
- [x] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [x] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [x] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/scripts).

# Others
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: openbb_terminal/cryptocurrency/quantitative_analysis/qa_controller.py [hunk 1]
```diff
@@ -260,16 +260,23 @@ def call_raw(self, other_args: List[str]):
             dest="descend",
             help="Sort in descending order",
         )
+        parser.add_argument(
+            "-s",
+            "--sortby",
+            help="The column to sort by",
+            type=str,
+            dest="sortby",
+        )
```
Comment (jmaslek) [inline openbb_terminal/cryptocurrency/quantitative_analysis/qa_controller.py:269]: choices?

**Comment by jmaslek** [comment]
sortby is not in autocomplete

**Comment by jmaslek** [comment]
Just using --descend without a sort col should probably sort the index?

Screen Shot 2022-11-01 at 9 27 19 AM

**Comment by jmaslek** [comment]
Functionally works on the stocks menu!  
Screen Shot 2022-11-01 at 9 28 48 AM

Economy:
```
2022 Nov 01, 09:29 (🦋) /economy/ $ qa

Please load a dataset before moving to the qa menu
```

Can probably be opened as a new bug, but in economy -- we can merge/concat all the dictionaries into a single df so that we can do raw and see all.

**Comment by jmaslek** [comment]
Crypto shows an Adj Close  - but not sortable:
```
2022 Nov 01, 09:31 (🦋) /crypto/qa/ $ raw adjclose

The following args couldn't be interpreted: ['adjclose']
                                                 Raw Data                                                  
┏━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━┓
┃            ┃ Open     ┃ High     ┃ Low      ┃ Close    ┃ Adj Close ┃ Volume          ┃ Returns ┃ Logret ┃
┡━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━┩
│ 2022-10-12 │ 1279.732 │ 1303.075 │ 1277.527 │ 1294.906 │ 1294.906  │ 8355638578.000  │ 0.012   │ 0.012  │
├────────────┼──────────┼──────────┼──────────┼──────────┼───────────┼─────────────────┼─────────┼────────┤
```

**Comment by colin99d** [comment]
@jmaslek I got your requested changes in. One issue is that the default sortby of index ascending means that users see the oldest dates. What do you want me to do with this?

**Comment by jmaslek** [comment]
> @jmaslek I got your requested changes in. One issue is that the default sortby of index ascending means that users see the oldest dates. What do you want me to do with this?

Lets leave as is for now

**Comment by jmaslek** [comment]
From the autocomplete options:

Screen Shot 2022-11-01 at 4 25 46 PM

Isnt --sortby the convention instead of --sort?  I am fine either way, just want the same
Screen Shot 2022-11-01 at 4 25 59 PM

**Comment by colin99d** [comment]
> From the autocomplete options:
> 
> Screen Shot 2022-11-01 at 4 25 46 PM
> 
> Isnt --sortby the convention instead of --sort? I am fine either way, just want the same Screen Shot 2022-11-01 at 4 25 59 PM

Updated this.

**Comment by colin99d** [comment]
@jmaslek I think it makes since to use all str.lower here as well, do you agree?

**Comment by jmaslek** [comment]
> @jmaslek I think it makes since to use all str.lower here as well, do you agree?

yes

**Review by jmaslek — APPROVED** [review]

**Pull request merged**